base/server/src/com/netscape/cms/profile/def/SignedCertificateTimestampListExtDefault.java

```diff
@@ -86,22 +87,12 @@ public String getText(Locale locale) {
     public void populate(IRequest request, X509CertInfo info)
             throws EProfileException {
         String method = "SignedCertificateTimestampListExtDefault: populate:";
-        CertificateExtensions inExts = null;
 
-        ObjectIdentifier ct_poison_oid = new ObjectIdentifier(CT_POISON_OID);
-        Extension ct_poison_ext = null;
-        try (DerOutputStream out = new DerOutputStream()) {
-            out.putOctetString(CT_POISON_DATA);
-            ct_poison_ext = new Extension(ct_poison_oid, CT_POISON_CRITICAL, out.toByteArray());
-            logger.debug(method + " ct_poison_ext id = " +
-            ct_poison_ext.getExtensionId().toString());
-            addExtension(CT_POISON_OID, ct_poison_ext, info);
-        } catch (IOException e) {
-//cfu
-            logger.debug(method + e.toString());
+        try {
+            CryptoUtil.addCTpoison(info);
         } catch (Exception e) {
-//cfu
-            logger.debug(method + e.toString());
+            logger.debug(method + "addCTpoison failed");
```

Comment:
We are catching a generic exception, which is not a good practice. We should first catch the known exception and then finally try catching the generic exception (see my next comment)

````
} catch (X509ExtensionException e) {
...
} catch (Exception e) {
...
}